sources/packages/api/src/database/entities/education-program.model.ts

```diff
@@ -198,4 +198,124 @@ export class EducationProgram extends RecordDataModel {
     type: "enum",
   })
   programIntensity: ProgramIntensity;
+
+  /**
+   * Institution program code of the given program
+   */
+  @Column({
+    name: "institution_program_code",
+  })
+  institutionProgramCode?: string;
+
+  /**
+   * Minimum hours check for a general program.
+   */
+  @Column({
+    name: "min_hours_week",
+  })
+  minHoursWeek?: string;
+
+  /**
+   * Identifier for aviation program.
+   */
+  @Column({
+    name: "is_aviation_program",
+  })
+  isAviationProgram?: string;
+
+  /**
+   * Minimum hours check for a aviation program.
+   */
+  @Column({
+    name: "min_hours_week_avi",
+  })
+  minHoursWeekAvi?: string;
+
+  /**
+   * Entrance eligibility of minimum grade 12
+   */
+  @Column({
+    name: "min_high_school",
+  })
+  minHighSchool?: boolean;
+
+  /**
+   * Entrance eligibility established by institution.
+   */
+  @Column({
+    name: "requirements_by_institution",
+  })
+  requirementsByInstitution?: boolean;
+
+  /**
+   * Entrance eligibility by B.C ITA.
+   */
+  @Column({
+    name: "requirements_by_bcita",
+  })
+  requirementsByBcita?: boolean;
```

Comment:
As per the agreement, acronyms are uppercase.

Reply:
BCITA is now in uppercase